osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -7,6 +7,7 @@
     {
       "tag": "CLOCK",
       "description": "Clock",
+      "note": null,
```

Comment:
Dit veld is optioneel neem ik aan, is het handig om dit overal op null te zetten?

Reply:
Voor de volledigheid had ik deze toegevoegd maar is inderdaad optioneel.